frontend/src/components/Pages/MissionPage/MissionHeader/MissionHeader.tsx

```diff
@@ -28,17 +28,15 @@ const TitleSection = styled.div`
 const InfoSection = styled.div`
     display: flex;
     flex-wrap: wrap;
-    gap: 8px;
-    max-width: 950px;
-`
-const StyledCard = styled(Card)`
-    display: flex;
-    flex: 1 0 0;
-    padding: 8px 16px;
-    flex-direction: row;
-    background: ${tokens.colors.ui.background__default.hex};
-    gap: 24px;
-    align-items: stretch;
+    gap: 32px;
+    width: fit-content;
+    @media (max-width: 600px) {
+        display: grid;
+        grid-template-columns: repeat(3, calc(75vw / 3));
+        gap: 32px;
+        width: fit-content;
+        align-items: end;
+    }
```

Comment:
<img width="1013" alt="image" src="https://github.com/user-attachments/assets/b41b3103-6dd8-4bdc-8838-56972604c0eb">
Should info section be same width as rest? Also it doesn't seem centered with regards to height